hardware/Teleinfo.cpp

```diff
@@ -292,6 +293,15 @@ void Teleinfo::MatchLine()
 				m_p3power.powerusage1 = ulValue;
 			break;
 		case TELEINFO_TYPE_PTEC:
+			if (vString.substr (0,2) == "HC")
+                        {
+                                SendSwitch(5, 1, 255, true, 0, "Heures Creuses");
```

Comment:
Please change the whole block (if/else) to

SendSwitch(5, 1, 255, (vString.substr (0,2) == "HC"), 0, "Heures Creuses");